### PR TITLE
Update API index

### PR DESCRIPTION
**Pull request by tpoomlmly**
This is Will Barber's edit from before he had to go.

### CODE REVIEW

**Comment by tpoomlmly** [comment]
just rebased on top of master

File: content/api/_index.md [hunk 1]
```diff
@@ -26,53 +26,31 @@ defined as `r`.
...
 Python already comes with plenty of [built-in
 libraries](https://docs.python.org/3.9/py-modindex.html) to use. We
 install some extra ones which may be of use:
 
-- [numpy 1.19.3](https://pypi.org/project/numpy/1.19.3/)
-- [matplotlib 3.3.3](https://pypi.org/project/matplotlib/3.3.3/)
-- [pandas 1.1.4](https://pypi.org/project/pandas/1.1.4/)
-- [scikit-learn 0.23.2](https://pypi.org/project/scikit-learn/0.23.2/)
-- [scipy 1.5.4](https://pypi.org/project/scipy/1.5.4/)
+- [numpy](https://pypi.python.org/pypi/numpy)
```
Comment (RealOrangeOne) [inline content/api/_index.md:49]: Putting the versions in here explicitly is probably a good idea, easier to read.

Also the link here isn't right, it immediately redirects.

Comment (tpoomlmly) [reply content/api/_index.md]: fixed

File: content/api/_index.md [hunk 2]
```diff
@@ -26,41 +26,19 @@ defined as `r`.
...
-r.power_board.serial
->>> 'SRO-AA2-7XS'
-r.servo_board.serial
->>> 'SRO-AA4-LG2'
-r.motor_board.serial
->>> 'SRO-AAO-RV2'
-```
+A log file is saved in the folder so you can see what your robot did,
+what it didn't do, and any errors it raised. Each log file name includes
+the date and time so you can look back at your previous runs.
```
Comment (RealOrangeOne) [inline content/api/_index.md:41]:
```suggestion
A log file is saved in the `output/` folder so you can see what your robot did,
what it didn't do, and any errors it raised. Each log file is in a folder based on its
date and time, so you can look back at your previous runs.
```

**Review by jefftheprogrammer — APPROVED** [review]

**Pull request merged**